### DISCUSSION TITLE
Expected yarn version 4.0.0-rc.34

### DISCUSSION THREAD

**Discussion by leookun** [flex-development/mlly#310] [Q&A]
Greate tools! thanks~
It reported a version error  when installing  with yarn :

```sh
 @flex-development/is-builtin@2.0.0: The engine "yarn" is incompatible with this module. Expected version "4.0.0-rc.34". Got "1.22.19"
```
Really want to install it with yarn@^1

**Comment by unicornware** [comment]
@leookun sorry i'm just seeing this! 😅

i'm thinking this error is because i have a [`yarn` key under `engines`][1] in that project. if you're still facing this issue, can you try installing with `yarn --ignore-engines`?

[1]: https://github.com/flex-development/is-builtin/blob/2.0.0/package.json#L165
